packages/tooltip/src/context.ts

```diff
@@ -7,7 +7,11 @@ export interface TooltipActionsContextData {
         position: [number, number],
         anchor?: TooltipAnchor
     ) => void
-    showTooltipFromEvent: (content: JSX.Element, event: MouseEvent, anchor?: TooltipAnchor) => void
+    showTooltipFromEvent: (
```

Comment:
Seems like `MouseEvent | TouchEvent` would me more accurate, what do you think?

Reply:
Well, technically it means a logic change, but it can be tidier to support unwrapping the client coords inside the function. 

(MouseEvent has one clientX/Y at the top level, while TouchEvent supports multi-touch, so all of the clientX/Y coords are in an array of touches)

Patched in https://github.com/WilliamABradley/nivo/commit/4ad5f67ffb61af4f8d5cb29b18fd55c0259a6ab3